docs/src/main/sphinx/develop/group-provider.md

```diff
@@ -20,18 +20,22 @@ must be wrapped as a Trino plugin and installed on the cluster.
 
```

Comment:
This is a section that helps to implement a group provider. What about a sections where we tell users how to use group providers? Is there such section? If so let's update it.

CC: @mosabua